script.js

```diff
@@ -97,34 +127,54 @@ const render = () => {
 
 const onChangeCheckbox = (index) => {
   allTasks[index].isCheck = !allTasks[index].isCheck;
-  localStorage.setItem("tasks", JSON.stringify(allTasks));
   render();
 };
 
-const removeTasks = (index) => {
+const removeTasks = async (item, index) => {
   allTasks.splice(index, 1);
-  localStorage.setItem("tasks", JSON.stringify(allTasks));
+  console.log(item.id);
+
+  const resp = await fetch(`http://localhost:8000/deleteTask?id=${item.id}`, {
+    method: "DELETE",
+    headers: {
+      "Content-Type": "application/json;charset=utf-8",
+      "Access-Control-Allow-Origin": "*",
+    },
+  });
+  let result = await resp.json();
+  allTasks = result.data;
   render();
 };
 
 const editTasksFunction = (index) => {
   editTasks = index;
   inputResult = allTasks[index].text;
-  localStorage.setItem("tasks", JSON.stringify(allTasks));
   render();
 };
 
-const saveEditFunction = (index) => {
+const saveEditFunction = async (index) => {
   allTasks[index].text = inputResult;
   editTasks = null;
   inputResult = "";
-  localStorage.setItem("tasks", JSON.stringify(allTasks));
+
+  const resp = await fetch("http://localhost:8000/updateTask", {
+    method: "PATCH",
+    headers: {
+      "Content-Type": "application/json;charset=utf-8",
+      "Access-Control-Allow-Origin": "*",
+    },
+    body: JSON.stringify({
+      id: allTasks[index].id,
+      text: inputResult
+    }),
+  });
+  let result = await resp.json();
+  inputResult = result;
```

Comment:
why need this row?

Reply:
i was think, what is this right. i think what i return a result from response fetch method PATCH. isn`t? I need remove this row? because it`s work without this rows.